development/code-guidelines/tasks-metatasks-guidelines.md

```diff
@@ -0,0 +1,138 @@
+# Tasks and MetaTasks Coding Guidelines
+
+## Introduction
+
+OSC Takss and metatask classes are used to synchronize information across different systems. This document outlines the code conventions and guidelines that must be followed when authoring tasks and metatasks. 
```

Comment:
typo tasks. 

OSC Tasks and metatasks classes are used for the core orchestration functions and for other long running asynchronous operations within OSC.